BaseApi/terraform/production/main.tf

```diff
@@ -60,3 +60,38 @@ module "development" {
   task_definition_secrets      = {}
   task_definition_secret_count = number # This number needs to reflect the number of environment variables provided
 }
+
```

Comment:
This file will stay the same as it is, perhaps we can just delete the 'development' module from it as this is just the file specifying the provider and setting up the s3 state (so the commented out puts are not required). This applies to the other main.tf files as well.

Reply:
I will remove all change I made to the main.tf files to leave them as they were.